flang/tools/bbc/bbc.cpp

```diff
@@ -279,22 +279,28 @@ static mlir::LogicalResult convertFortranSourceToMLIR(
   } else {
     // run the default canned pipeline
     pm.addPass(std::make_unique<Fortran::lower::VerifierPass>());
+
+    // simplify the IR
```

Comment:
Are these change supposed to be part of this PR ?
I am not against these changes, just want to make sure they are not pushed by accident since they do seem connected to TODO.

Reply:
Bonus changes to keep bbc and tco in sync.